src/scss/settings/_variables.scss

```diff
@@ -21,15 +21,15 @@ $type: (
         default: (14, 18)
     ),
     mid: (  // h3, h4, h5, h6 sizes (differentiated by font and weight not size)
-        default: (16, 24)
+        default: (20, 24)
     ),
     large: ( // h2
-        default: (20, 28),
-        narrow: (18, 24)
+        default: (24, 28),
+        narrow: (20, 24)
     ),
     jumbo: ( // h1
-        default: (24, 32),
-        narrow: (20, 24)
+        default: (30, 40),
+        narrow: (24, 24)
```

Comment:
Worth testing all of these new combinations of font-sizes and line-height's.

Just tried testing this one myself - having the same font-size and line-height reads terribly when the text does wrap onto 2 lines, which is pretty likely on mobile.  Would be worth asking Bruno what line-height this should be set to instead as looks like an oversight